kolibri/plugins/user_profile/assets/src/views/ChangeFacility/ConfirmMerge.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+
+  <div>
+    <h1>{{ $tr('documentTitle') }}</h1>
+    <p>{{ $tr('confirmMergeInfoLine') }}</p>
+    <div class="confirm">
+      <KCheckbox
+        ref="kCheckbox"
+        :label="$tr('consequences')"
+        :checked="false"
+        @change="isConfirmed = !isConfirmed"
+        @keydown.enter="handleContinue"
+      />
+    </div>
+
+    <BottomAppBar>
+      <slot name="buttons">
+        <KButtonGroup>
+          <KButton
+            :primary="false"
+            :text="coreString('backAction')"
+            appearance="flat-button"
+            data-test="backButton"
+            @click="sendBack"
+          />
+          <KButton
+            :primary="true"
+            :disabled="!isConfirmed"
+            :text="coreString('continueAction')"
+            data-test="continueButton"
+            @click="handleContinue"
+          />
+        </KButtonGroup>
+      </slot>
+    </BottomAppBar>
+
+  </div>
+
+</template>
+
+
+<script>
+
+  import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
+  import BottomAppBar from 'kolibri.coreVue.components.BottomAppBar';
+  import { inject, ref } from 'kolibri.lib.vueCompositionApi';
+
+  export default {
+    name: 'ConfirmMerge',
+    metaInfo() {
+      return {
+        title: this.$tr('documentTitle'),
+      };
+    },
+    components: { BottomAppBar },
+
+    mixins: [commonCoreStrings],
+
+    setup() {
+      const changeFacilityService = inject('changeFacilityService');
+      const isConfirmed = ref(false);
+
+      function handleContinue() {
+        if (isConfirmed.value)
+          changeFacilityService.send({
```

Comment:
sure, I'm all ears, I will change it if people think it's better (I am so used to it since so many years that I see it as readable or more than using curly brackets) Anwyay, if we decide to use that convention I think we should add a linting rule to detect it. I'm pretty sure you'll find it everywhere in any of the libs we use, as this is a common use in all the languages with syntax derived from C.

> Also, I have one question about keyboard vs. mouse navigation:
> (1) When using mouse, when I check on the checkbox and then press enter on keyboard, it takes us immediately to another step, whereas (2) when using keyboard, when I check the checkbox with space and then press enter on keyboard, nothing happens and I need to tab to "Continue" button and press enter again to get to the next state. Was this discrepancy intentional? I am actually not sure whether behavior (1) or (2) is preferred.

No, it was not intentional, actually I didn't even know it was happening. I prefer (1) because it seems more agile, but I'm not sure that can be achieved just using the keyboard as the events will overlap. Any suggestion?

Reply:
I must admit, I am also not a huge fan of the single line of code following an `if` statement - and also prefer requiring an expression block.

Seems like we can enforce this via eslint's `curly` rule, with the `all` setting: https://eslint.org/docs/latest/rules/curly